locales/en-US.yml

```
@@ -2108,23 +2108,20 @@ en-US:
         tomorrow: New episode airs tomorrow
   images:
     confirm_delete: Are you sure you want to delete this image?
-    upload_criteria:
-      '1': A maximum resolution of 2000x3000
-      '2': A minimum resolution of 500x750
-      '3': Aspect ratio of 1:1.5 (2:3)
-      '4': A maximum resolution of 3840x2160
-      '5': A minimum resolution of 400x225
-      '6': Aspect ratio of 16x9 (1.78:1) and 4:3 images are supported
-      '7': A minimum resolution of 1280x720
-      '8': Aspect ratio of 16x9 (1.78:1)
-      '9': A minimum resolution of 300x450
-      backdrops_must_meet: 'Backdrops must meet the following criteria:'
-      episode_stills_must_meet: 'Episode stills must meet the following criteria:'
-      headshots_must_meet: 'Headshots must meet the following criteria:'
-      posters_must_meet: 'Posters must meet the following criteria:'
-      profile_must_meet: 'Profile images must meet the following criteria:'
+    uploader:
+      aspect_ratio: Aspect ratio of %{ratio}
+      aspect_ratio_both: Aspect ratio of %{ratio_first} and %{ratio_second} images are supported
```

```suggestion
      aspect_ratio_both: Aspect ratio of %{ratio_first} or %{ratio_second}
```

This key isn't really necessary, either way the wording should be cleaned up to match the rest.

---

ok